component-samples/demo/aio/docker-compose.yml

```diff
@@ -22,8 +22,10 @@ services:
       - service.env
     volumes:
       - ./app-data:/home/fdo/app-data 
+    extra_hosts:
+     - "host.docker.internal:host-gateway"
     mem_limit: 500m
     mem_reservation: 200m
     cpu_shares: 1024
     pids_limit: 500
-    network_mode: host
+
```

Comment:
No.   When we remove network_mode: host then we are already using a bridge network.  We are just using the "default" created bridge network.  All services can take to each other over the "default" bridge network but hey would have to use the IP address assigned by the bridge network instead of using "localhost"/127.0.0.1.     The  issue  is we must move away from using "localhost" and 127.0.01 in any urls regardless.  Creating a custom bridge won't solve that.